Makefile

```diff
@@ -24,6 +24,10 @@ AUTHORS: .mailmap .git/HEAD
 version/version.go:
 	./version/version.sh > $@
 
+${PREFIX}/bin/garbagecollect: version/version.go $(shell find . -type f -name '*.go')
```

Comment:
In the `registry` package there is a cobra command for the main registry command. Add a subcommand called `gc` that does the garbage collection. There is no reason to have a separate binary.

Please do this so we can move forward with this PR.